6_3_iterable.js

```diff
@@ -0,0 +1,48 @@
+class Group {
+    constructor(array) {
+        this.array = array;
+    }
+
+    static from(array) {
+        let group = new Group(array);
+        return group;
+    }
+
+    add(value) {
+        this.array.includes(value) || this.array.push(value);
```

Comment:
I would like to replace it with `!this.array.includes(value) && this.array.push(value)`.

Reply:
And maybe we should return some information if i'ts added correctly or not